pkg/codegen/python/gen.go

```diff
@@ -2151,8 +2151,12 @@ func genPackageMetadata(
 		fmt.Fprintf(w, "          'Repository': '%s'\n", pkg.Repository)
 		fmt.Fprintf(w, "      },\n")
 	}
-	if pkg.License != "" {
-		fmt.Fprintf(w, "      license='%s',\n", pkg.License)
+	// Remind the user to provide a license if none is provided in
+	// the schema.
+	if license, err := pkg.GetLicense(); err == nil {
```

Comment:
I guess this isn't unique to Python? Should this just be an error when binding a schema?

Reply:
Yes, probably!